metadata/service/exporter/configurable/exporter.go

```diff
@@ -84,6 +85,22 @@ func (exporter *MetadataServiceExporter) Export(url *common.URL) error {
 	return nil
 }
 
+func getMetadataPort() string {
+	rootConfig := config.GetRootConfig()
+	port := rootConfig.Application.MetadataServicePort
+	if port == "" {
+		protocolConfig, ok := rootConfig.Protocols[constant.DefaultProtocol]
+		if ok {
+			port = protocolConfig.Port
+		} else {
+			logger.Warnf("[Metadata Service] Dubbo-go %s version's MetadataService does not support triple protocol,"+
```

Comment:
The warning message would better be "[Metadata Service] Dubbo-go %s version's MetadataService only support dubbo protocol, ".  Because it might be other protocols beyond dubbo and triple.

Reply:
Yes, that's better